### PR TITLE
add a 404 page (#4)

### LINKED ISSUE #4: Add a 404 page
Create a custom 404 page.

### PR DESCRIPTION
**Pull request by tusharnankani**
Closes #4 

Looks like:

image

Tested on all devices: 

image

### CODE REVIEW

File: public/html/404.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+		<meta property="og:type" content="website" />
+		<meta
+			property="og:description"
+			content="The OTC CatchUp 404 Page."
+		/>
+		<meta property="og:url" content="https://catchup.ourtech.community" />
+		<meta
+			property="og:image"
+			content="https://catchup.ourtech.community/public/img/home.png"
+		/>
```
Comment (tusharnankani) [inline public/html/404.html:24]: Hoping, we don't need a custom 404 `og:image`.

Comment (HarshKapadia2) [reply public/html/404.html:24]: No we won't, but that's a good realisation. Thank you! 👍

**Comment by tusharnankani** [comment]
Issue with CSS & JS while rendering `/summary/404`

Screenshot (837)

**Comment by tusharnankani** [comment]
Deploy checks are [failing](https://github.com/OurTechCommunity/catchup/runs/4607271968?check_suite_focus=true): 

image

File: public/html/404.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+		<script defer src="public/js/script.js"></script>
+
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+			rel="stylesheet"
+		/>
+
+		<link rel="stylesheet" href="/public/css/style.css" />
```
Comment (tusharnankani) [inline public/html/404.html:42]: Fixed it by changing `href` from `"public/css/style.css"` to `"/public/css/style.css"`
Thanks to [this](https://stackoverflow.com/questions/14277936/404-page-not-displaying-styles#:~:text=So%20you%20should,css/reset.css%22%3E).

**Comment by tusharnankani** [comment]
Rendering now 👍🏻

image

Only scared when we reach the 404th session haha.

File: public/html/404.html (hunk 2 above)
Comment (HarshKapadia2) [reply public/html/404.html:42]: Oh wow this was a good fix. Thank you!

**Comment by HarshKapadia2** [comment]
Haha yes I was going to message you about correcting the other links as well. Good fix.
(I was looking at redirecting to the 404 page. IDEK how I would've done that LOL. Your fix is much better.)

Just FYI if you didn't already know: Next time this happens where you forget to do something, you can interactively rebase your PR and squash the commit into one and force push. This depends from repo to repo, so ask the maintainer before rebasing. I don't care, so it's okay. Some repos require you to have one commit per PR no matter what, so it depends. This was just to let you know.

Also, I think you have some formatting issues, but I'll push a commit to correct that.

Thank you so much for a PR, @tusharnankani!

**Pull request merged**